### ISSUE TITLE
Environment Config from radix-cli deploy

### ISSUE THREAD

**Issue by soofstad** [equinor/radix-platform#758]
**My problem**

I have a problem coming up with a practical way of defining which version (container image tag) of the different components should be deploy in a "deploy-only" radix deploy job.

**Describe the solution you'd like**

Have `radix-cli` take multiple key-value pair of "variables" that takes precedence over variables defined in `radixconfig.yaml`.
"variables" in `radixconfig.yaml` could act as "fallback/default".

Example;
Given `radixconfig.yaml`:
```yaml
  components:
    - name: web
      image: datamodelingtool.azurecr.io/dmt/web:{webImageTag}
      environmentConfig:
        - environment: prod
          webImageTag: production
        - environment: dev
          webIimageTag: latest
```

And a Github actions step like so;

```yaml
      - name: Deploy on Radix
        uses: equinor/radix-github-actions@master
        env:
          APP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.APP_SERVICE_ACCOUNT_TOKEN }}
        with:
          args: >
            create job
            deploy
            --application data-modelling-tool
            --environment ${{ inputs.radix-environment }}
            --user stoo@equinor.com
            --context production
            --from-config
            --token-environment
            --follow
            --var webImageTag=1.2.3
```
A new deployment of the web component will be made with the "1.2.3" tag of the container image.

**Describe alternatives you've considered**
There exists two workarounds that I now of today:
 - Re-tagging the new version with the pre-defined tag used for the different environments (this causes issues when rolling back)
 - First making a commit updating the pre-defined values in the `radixconfig.yaml` file, then running the radix-ci action. (This requires having CI with write access to the repository, and creates a messy git log)

**Comment by emirgens** [comment]
Our ref: [263993](https://dev.azure.com/equinor/Radix/_workitems/edit/263993)

**Comment by oyron** [comment]
After some discussion, we have decided to implement this in one of the coming sprints. We will introduce the ability to pass the imageTag from the GitHub action (but not a generic variable passing concept).

**Comment by oyron** [comment]
The feature to be implemented is now registered in issue #911 
Progress will be reported there. Closing this issue.
